src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitFormIncident.js

```diff
@@ -0,0 +1,106 @@
+import React, { useCallback, useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Heading } from '@datapunt/asc-ui';
+
+import { typesList, priorityList } from 'signals/incident-management/definitions';
+
+import { subcategoriesType } from 'shared/types';
+
+import TextArea from 'components/TextArea';
+
+import { StyledButton } from './styled';
+
+import IncidentSplitRadioInput from './IncidentSplitRadioInput';
+import IncidentSplitSelectInput from './IncidentSplitSelectInput';
+
+export const INCIDENT_SPLIT_LIMIT = 10;
+
+const IncidentSplitFormIncident = ({ parentIncident, subcategories, register }) => {
```

Comment:
`subcategories` is here just passed through. A context construction would be beter in this case.

Reply:
I hear what you are saying, but it will also make the underlying components less flexible so harder to adopt them for more general use in the future.

I would prefer to use a `useCategories` where needed, so we don't need to pass anything and also don't need a custom context just for the sake of passing data around, besides that when we would use the `useContext` hook for this we will rerender child components more then needed.